.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "markdownlint.config": {
+        "MD028": false,
+        "MD025": {
+            "front_matter_title": ""
+        }
+    }
+}
```

Comment:
Can you please remove this from the files?

Reply:
@marvinbuss - Good catch - deleted file from PR.